keras/src/callbacks/backup_and_restore.py

```diff
@@ -90,6 +90,12 @@ def __init__(
         self._training_metadata_path = file_utils.join(
             backup_dir, "training_metadata.json"
         )
+        self._prev_weights_path = file_utils.join(
+            backup_dir, "_latest.weights.h5"
```

Comment:
This should probably be called something like `_backup`, make it explicit

Reply:
done